.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,6 @@
+---
+name: How-to and other issues
+about: Please, post other questions on StackOverflow with `gpflow` tag.
+---
+
+**For questions on how to work with GPflow, or support for problems that are not verified bugs in GPflow, please go to [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow) and use the [gpflow tag](https://stackoverflow.com/questions/tagged/gpflow) for your question.** This makes it easier for other users of GPflow to help you with your problem, and also makes your question and any answers more easily discoverable for other users who may come across the same question in the future!
```

Comment:
what do you mean by verified bugs? don't we want users to post _all_ bugs here? I don't think SO is a good place for bug reports, partly cos that's not really what it's meant for, and partly cos we'll just end up loudly publicising our history of bugs, which looks bad

Reply:
Yeah we should probably edit that. I took it from an old version, but I agree that we should clear it up! Suggestions welcome:)

Reply:
```suggestion
**GitHub issues are most helpful for reporting issues that may require changes to the GPflow code base itself. If your question is about how to *use* GPflow, please go to [StackOverflow](https://stackoverflow.com/questions/tagged/tensorflow) and use the [gpflow tag](https://stackoverflow.com/questions/tagged/gpflow) for your question.** This makes it easier for other users of GPflow to help you with your problem, and also makes your question and any answers more easily discoverable for other users who may come across the same question in the future! If your question is about an omission or lack of clarity in the GPflow documentation, please use the documentation issue template.
```
How about this, @joelberkeley-pio ?

Reply:
great!